module/VuFind/src/VuFind/Connection/ExternalVuFind.php

```diff
@@ -55,6 +57,14 @@ class ExternalVuFind implements
      */
     protected $baseUrl = null;
 
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->cacheOptionsSection = 'ExternalVuFind';
```

Comment:
I feel like I should document somewhere that you need a config.ini section Config_ExternalVuFind to make this work, i.e. the following, but not sure where to note that.
```
[Cache_ExternalVuFind]
disabled = false
ttl = 30
```

Reply:
Good call, did both.